src/utils/error/error.utils.ts

```diff
@@ -1,3 +1,17 @@
 export function isError(data: unknown): data is Error {
   return data instanceof Error;
 }
+
+export enum ErrorTypes {
+  Error = "Error",
+  RangeError = "RangeError",
+  ReferenceError = "ReferenceError",
+  SyntaxError = "SyntaxError",
+  URIError = "URIError",
+  PageNotFound = "Page not found",
+  SettingsTypeNotFound = "Settings type not found",
```

Comment:
I would avoid labelling all of them as "not found", as there are some subtle differences in these ones:

- `SettingsTypeNotFound`: I would do something like `throw new Error(settings.type ? ErrorTypes.MissingSettingsType : ErrorTypes.UnexpectedSettingsType)`. It would even be better if you throw a custom error where we can add additional details (the actual value we got, the expected value(s)...) but that's probably overkill right now.
- `TxIdNotFound`: I would rename this one to `MissingTxId`.

Like this they convey a bit more information:
- "Not found": Something was queried/mapped/loaded but was not there.
- "Missing": Something should have been provided/set, but wasn't.
- "Unexpected": Something was provided, but the value wasn't right.